alluxio/alluxio_file_system.py

```diff
@@ -141,20 +143,32 @@ def __init__(
                 "Supply either 'etcd_hosts' or 'worker_hosts', not both"
             )
         self.logger = logger or logging.getLogger("AlluxioFileSystem")
+        if etcd_hosts and not worker_hosts:
+            self.logger.warning(
+                "Does not supply 'etcd_host'. Cannot tolerate Alluxio cluster changes."
+            )
         self.session = self._create_session(concurrency)
 
         # parse options
         page_size = ALLUXIO_PAGE_SIZE_DEFAULT_VALUE
+        hash_node_per_worker = int(ALLUXIO_HASH_NODE_PER_WORKER_DEFAULT_VALUE)
         if options:
             if ALLUXIO_PAGE_SIZE_KEY in options:
                 page_size = options[ALLUXIO_PAGE_SIZE_KEY]
                 self.logger.debug(f"Page size is set to {page_size}")
+            if ALLUXIO_HASH_NODE_PER_WORKER_KEY in options:
+                hash_node_per_worker = int(
```

Comment:
why not int?

Reply:
The const is all string format, the const file is kind of Alluxio property key files now